<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Terminal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(93, 142, 255);
background-color: rgb(115, 122, 126);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Collection</string>
    </property>
    <addaction name="actionCollectSetting"/>
    <addaction name="actionCollectStart"/>
    <addaction name="actionCollectEnd"/>
    <addaction name="actionScreenShot"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionFastCmd"/>
   <addaction name="actionSaveFile"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionCollectSetting"/>
   <addaction name="actionCollectStart"/>
   <addaction name="actionCollectEnd"/>
   <addaction name="actionScreenShot"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/SaveFile.png</normaloff>:/images/SaveFile.png</iconset>
   </property>
   <property name="text">
    <string>SaveFile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFastCmd">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/FastCmd.png</normaloff>:/images/FastCmd.png</iconset>
   </property>
   <property name="text">
    <string>FastCmd</string>
   </property>
   <property name="toolTip">
    <string>FastCmd</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionCollectStart">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/collectStart.png</normaloff>:/images/collectStart.png</iconset>
   </property>
   <property name="text">
    <string>CollectStart</string>
   </property>
  </action>
  <action name="actionCollectEnd">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/collectEnd.png</normaloff>:/images/collectEnd.png</iconset>
   </property>
   <property name="text">
    <string>CollectEnd</string>
   </property>
  </action>
  <action name="actionCollectSetting">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/collectSetting.png</normaloff>:/images/collectSetting.png</iconset>
   </property>
   <property name="text">
    <string>CollectSetting</string>
   </property>
  </action>
  <action name="actionScreenShot">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/screenshot.png</normaloff>:/images/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>ScreenShot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
